modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/legacy/definition_diagram_controller.js

```diff
@@ -123,16 +123,20 @@ AUI.add(
 				return fields;
 			},
 
-			serializeDefinition(json) {
+			serializeDefinition(json, draftVersion) {
```

Comment:
Hey @MariMalta, Can you please sort the parameters alphabetically? Thx

Reply:
Yes, I'm going to do it